src/activities/number_click/Number_click.qml

```diff
@@ -0,0 +1,604 @@
+                radius: 60
+
+                anchors.horizontalCenter: parent.horizontalCenter
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: parent.height * 0.05
+
+                Image {
+                    id: buttonTick
+                    source: Activity.url + "tick.svg"
+
+                    height: parent.height * 0.6
+                    fillMode: Image.PreserveAspectFit
+
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    anchors.verticalCenter: parent.verticalCenter
+                }
+
+                MouseArea {
+                    id: buttonArea
+
+                    anchors.fill: parent
+                    hoverEnabled: true
+
+                    onEntered: { parent.color = "#00D022" }
+                    onExited: { parent.color = "#00A31B" }
+                    onPressed: {
+                        buttonTick.height = parent.height * 0.7
+                        activity.audioEffects.play("qrc:/gcompris/src/core/resource/sounds/bleep.wav")
+                        Activity.checkAnswer()
+                    }
+                    onReleased: { buttonTick.height = parent.height * 0.6 }
+                }
+            }
+        }
+
+        /* Set the error panel */
+        Rectangle {
+            id: errorpanel
+
+            visible: false
+
+            height: parent.height * 0.6
+            width : parent.width * 0.6
+
+            color: "white"
+            border.width: 4
+            border.color: "black"
+            radius: 30
+
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.verticalCenter: parent.verticalCenter
+
+            MouseArea {
+                id: errorpanelArea
+
+                anchors.fill: parent
+                hoverEnabled: true
+
+                onEntered: {  }
+                onExited: {  }
+                onPressed: {
+                    activity.audioEffects.play("qrc:/gcompris/src/core/resource/sounds/bleep.wav")
+                    errorpanel.visible = false
+                }
+                onReleased: {  }
+            }
+
+            Rectangle {
+                id: correctAnswer
+
+                height: tenthbar.height * 1.05
+                width: firstbar.width * 1.25
+
+                anchors.verticalCenter: parent.verticalCenter
+                //anchors.horizontalCenter: firstbar.horizontalCenter
+
+                border.color: "green"
+                border.width: 7
+                radius: 10
+            }
+
+            Rectangle {
+                id: wrongAnswer
+
+                height: tenthbar.height * 1.05
+                width: firstbar.width * 1.25
+
+                anchors.verticalCenter: parent.verticalCenter
+                //anchors.horizontalCenter: tenthbar.horizontalCenter
+
+                border.color: "red"
+                border.width: 7
+                radius: 10
+            }
+
+            Rectangle {
```

Comment:
I haven't tested exactly, but you should use a Grid/Row/Column with Repeater instead of having 10 times the bars (for example: https://invent.kde.org/education/gcompris/-/blob/master/src/activities/algorithm/Algorithm.qml#L85-104)